libsrc/nes/Makefile.inc

```diff
@@ -1,8 +1,8 @@
-../tgi/nes-64-56-2.tgi: ../libwrk/nes/clrscr.o \
-                        ../libwrk/nes/cputc.o  \
-                        ../libwrk/nes/get_tv.o \
-                        ../libwrk/nes/gotoxy.o \
-                        ../libwrk/nes/popa.o   \
-                        ../libwrk/nes/ppu.o    \
-                        ../libwrk/nes/ppubuf.o \
-                        ../libwrk/nes/setcursor.o
+../target/nes/drv/tgi/nes-64-56-2.tgi: ../libwrk/nes/clrscr.o \
+                                   ../libwrk/nes/cputc.o  \
+                                   ../libwrk/nes/get_tv.o \
+                                   ../libwrk/nes/gotoxy.o \
+                                   ../libwrk/nes/popa.o   \
+                                   ../libwrk/nes/ppu.o    \
+                                   ../libwrk/nes/ppubuf.o \
+                                   ../libwrk/nes/setcursor.o
```

Comment:
Would it look better if the first "fold" was done at the colon (i.e., the "clrscr.o" path was on the second line)?  Then, the indent wouldn't need to be so wide.